db/seeds.rb

```diff
@@ -5,11 +5,46 @@
 ############################################################
 
 users = [
-    { :email => "ben@bitdiddle.com", :password => "bitsrocks" },
-    { :email => "generic@email.com", :password => "asdfasdf" },
-    { :email => "bob@schmitt.com", :password => "imboring" },
-    { :email => "genericasiankid@gmail.com", :password => "password" },
-    { :email => "rosenthal@policy.com", :password => "publicpolicy" }
+    {
+        :email => "ben@bitdiddle.com",
+        :password => "bitsrocks",
+        :username => "ben",
+        :avatar => "/path/to/ben.png",
+        :political_blurb => "I like Ron Bitdiddle",
+        :political_hero => "Ron Bitdiddle",
+        :political_spectrum => 2 },
```

Comment:
Tiny style nit: I think it looks better like
{
    :email => "etc@etc.com",
    ...more fields...
}
...but yea, it's not that important.